### PR TITLE
Adding breakdown of benefits

### LINKED ISSUE #50: Add Breakdown of Benefits and/or Credits in Visual Form

### PR DESCRIPTION
**Pull request by emieelvire**
Fixes #50

### CODE REVIEW

**Comment by emieelvire** [comment]
Screenshot 2024-04-11 at 10 05 52 AM
Current progress. Still trying to figure out how to get the components to place into the breakdown

**Comment by PavelMakarchuk** [comment]
run `git pull origin master`

**Comment by PavelMakarchuk** [comment]
Looks good, couple of comments:
- Could we format the table the same way as we have the first one, 
Program / Married / Not married / Delta / Delta %

File: app.py [hunk 1]
```diff
@@ -52,17 +54,26 @@ def get_programs(state_code, head_employment_income, spouse_employment_income=No
     situation["households"] = {
         "your household": {"members": members, "state_name": {YEAR: state_code}}
     }
-  
     simulation = Simulation(situation=situation)
 
-    simulation = Simulation(situation=situation)
```
Comment (emieelvire) [inline app.py:58]: removing this line because it seems redundant

**Comment by PavelMakarchuk** [comment]
need to resolve conflicts

**Pull request merged**